tests/collection.test.js

```diff
@@ -298,6 +299,78 @@ describe('User Transitions', () => {
     // Add validation for correct state changes
   })
 
+  test('SetMaxCommissionFeeBPS: Sets a new commison fee BPS', async () => {
+    const collectionContract = await zilliqa.contracts.at(collectionContractAddress)
+
+    const tx = await callContract(
+      accounts.contractOwner.privateKey,
+      collectionContract,
+      'SetMaxCommissionFeeBPS',
+      [
+        {
+          vname: 'new_max_commission_fee_bps',
+          type: "Uint128",
+          value: "3000"
+        }
+      ],
+      0,
+      false,
+      false
+    )
+    
+    console.log("SetMaxCommissionFeeBPS: Sets a new commison fee BPS" ,tx.receipt)
+    expect(tx.receipt.success).toEqual(true);
+
+    const txCreateCollection = await callContract(
+      accounts.stranger.privateKey,
+      collectionContract,
+      'CreateCollection',
+      [
+        {
+          vname: 'commission_fee',
+          type: "Uint128",
+          value: "3000"
+        }
+      ],
+      0,
+      false,
+      false
+    )
+    console.log("SetMaxCommissionFeeBPS: CreateCollection with 3000 fee" ,txCreateCollection.receipt)
```

Comment:
Can you also validate that the state of the contract has been updated?

Reply:
Thanks @StayFoolisj, Have updated the test case.